CLIENT-SPECIFICATION.md

```diff
@@ -151,13 +151,13 @@ It is possible that due to this page resolution logic, the client may show a pag
 
 #### If a page is not found
 
-If a page cannot be found in _any_ platform, then it is RECOMMENDED that clients display an error message with a link to create a new issue against the `tldr-pages/tldr` GitHub repository. Said link might take the following form:
+If a page cannot be found in _any_ platform, then it is RECOMMENDED that clients display an error message with a link to create a new issue against the `tldr-pages/tldr` GitHub repository. The said link might take the following form:
```

Comment:
The previous form is correct and AFAIK more common. If the form doesn't appeal to you, perhaps something different like "Such a link" would be preferable? (For the record, I'm fine with the existing wording.)

Reply:
```suggestion
If a page cannot be found in _any_ platform, then it is RECOMMENDED that clients display an error message with a link to create a new issue against the `tldr-pages/tldr` GitHub repository. Said link might take the following form:
```